test/lib/sdk/cppSdk.js

```diff
@@ -9,7 +9,7 @@ class CppSdk {
   constructor(version) {
     this.version = version;
     this.versionPath = getVersionPath('cpp', this.version);
-    this.sdkCppArchive = 'kuzzlesdk-cpp-amd64.tar.gz';
+    this.sdkCppArchive = 'kuzzlesdk-cpp-experimental-amd64.tar.gz';
```

Comment:
We will have a problem when we will start to use SDK builds from master branch instead of 1-dev.

Reply:
No problem, I'll just need to make a second `sdk-versions.yml` for master branch

Reply:
But not now and I'm not going to change again the package name

Reply:
Why duplicate the configuration file ? We could just name the SDK the same way in all branch. Furthermore we already have the branch name in the path

Reply:
How do you manage multiple versions in that case?

Reply:
We need to have explicit name with explicit version on our SDK. How you know you have donwload "experimental" SDK if they have all the same name?

Reply:
@Aschen : I think we need to merge this PR to fix documentation tests. I'll work on a manager to handle SDKs versions download.

Reply:
In this case it would be better to have the branch name in the sdk filename. The test framework will always execute the test against the master (latest) ou 1-dev branch I think